Tests/DatadogTests/Datadog/Mocks/LoggingFeatureMocks.swift

```diff
@@ -103,25 +105,51 @@ extension Log {
             tags: tags
         )
     }
+
+    static func mockRandom() -> Log {
+        return Log(
+            date: .mockRandomInThePast(),
+            status: .mockRandom(),
+            message: .mockRandom(),
+            error: .mockRandom(),
+            serviceName: .mockRandom(),
+            environment: .mockRandom(),
+            loggerName: .mockRandom(),
+            loggerVersion: .mockRandom(),
+            threadName: .mockRandom(),
+            applicationVersion: .mockRandom(),
+            userInfo: .init(id: .mockRandom(), name: .mockRandom(), email: .mockRandom(), extraInfo: [:]), // TODO: RUMM-1050 use `.mockRandom()`
+            networkConnectionInfo: .mockAny(), // TODO: RUMM-1050 use `.mockRandom()`
+            mobileCarrierInfo: .mockAny(), // TODO: RUMM-1050 use `.mockRandom()`
+            attributes: .mockAny(), // TODO: RUMM-1050 use `.mockRandom()`
+            tags: nil // TODO: RUMM-1050 use `.mockRandom()`
+        )
+    }
 }
 
-extension Log.Status {
+extension Log.Status: RandomMockable {
     static func mockAny() -> Log.Status {
         return .info
     }
+
+    static func mockRandom() -> Log.Status {
+        return [.debug, .info, .notice, .warn, .error, .critical].randomElement()!
+    }
```

Comment:
you can make `Log.Status: CaseIterable` and then `Log.Status.allCases.randomElement!` instead of manually maintaining this array

Reply:
`CaseIterable` implies SIL code generation and must be declared in the same file as `Log.Status`. I try to avoid enhancing production types with test utilities - to keep the compiled binary size small. But maybe it's too puristic way and we can do it as we do with `Equatable` in few places already 👍.

Reply:
Done 👍.